sources/core/objects/Human.cpp

```diff
@@ -509,7 +529,7 @@ void Human::Bump(IdPtr<IMovable> item)
 
         unsigned int blood_value = (GetRand() % 7) + 1;
 
-        if (IdPtr<Floor> floor = GetTurf())
+        if (IdPtr<Floor> floor = GetTurf()) // TODO Bleeding only with ballistic bullets
```

Comment:
You can change the probability of a blood creation depending on amount of brute damage.
```
const int ALWAYS_BLOOD_BORDER = 10;
int brute_helper = qMax(1, ALWAYS_BLOOD_BORDER - brute);
bool is_blood = ((GetRand() % brute_helper) == 1);
```

I think, a continuous bleeding is a little bit too much now - only some basic medical system should be implemented, because later on it will be reworked anyway.